wandb/jupyter.py

```diff
@@ -93,7 +93,8 @@ def notebook_metadata():
             list_running_servers()
         )  # TODO: sometimes there are invalid JSON files and this blows up
     except Exception:
-        logger.error(error_message)
+        if not silent:
+            logger.error(error_message)
```

Comment:
Ideally our `logger` would always be attached to a file so this would not need to know about "silent".  I think the real issue here is that in jupyter there are cases where the logger is connected to stdout.  If we keep this for now I would atleast add a comment saying we need to fix the logger issue.

Reply:
@KyleGoyette , please add the note that @vanpelt requests. the bug is that this is not using the logger initialized in wandb.init() but your solution is a good stop gap.

Also please fix the merge conflicts